bug/snapshot.go

```diff
@@ -4,24 +4,25 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/MichaelMure/git-bug/commentary"
 	"github.com/MichaelMure/git-bug/entity"
-	"github.com/MichaelMure/git-bug/identity"
 )
 
 // Snapshot is a compiled form of the Bug data structure used for storage and merge
 type Snapshot struct {
 	id entity.Id
 
-	Status       Status
-	Title        string
-	Comments     []Comment
-	Labels       []Label
-	Author       identity.Interface
-	Actors       []identity.Interface
-	Participants []identity.Interface
-	CreateTime   time.Time
+	status         Status
+	title          string
+	description    commentary.Comment
+	comments       []commentary.Comment
+	labels         []Label
+	authorID       entity.Id
+	actorIDs       []entity.Id
+	participantIDs []entity.Id
```

Comment:
Unless it is required to display all users which participated on per bug basis, a bug shouldn't hold the references to identities.
At the moment this seems to be not required. Instead move this data over the identities.
E.g. an identity has three lists which contain bugIDs of all authored, interacted and participated bugs.